doc/developer/webservices.rst

```diff
@@ -240,3 +289,26 @@ Result
 
 The 'Content-Type' header is 'text/csv',
 and the data contains the given 'csv' data.
+
+Geometry operations
+===================
+
+This service gives geometry operations (actually only one)
+
+Difference
+----------
+
+URL: ``.../difference``
+
+Method: ``POST``
+
+Data:
+
+.. code:: json
+
+   {
+       "geom": <The geom to extrude>
+       "diff": <The geom to extrude with>
```

Comment:
on Sqlite it's geom1, geom2
on argis, geometry1, geometry2
on OGC: g1, g2 (http://portal.opengeospatial.org/files/?artifact_id=25354)

then I can put g1, g2 event if I think that's worth ...